packages/baby-jubjub/tests/index.test.ts

```diff
@@ -34,7 +35,7 @@ describe("BabyJubjub", () => {
 
         const expectedPackedPoint = babyjub.packPoint(publicKey)
```

Comment:
Might make sense to convert this to a bigint (using Scalar.fromRprLE) and compare as bigints.  Not really different in functionality, but might better match the intent that the output should be "the same number" with each library converting in its own preferred way.

The direct use of Scalar.fromRprLE makes this more awkward than ideal, so this is a weak suggestion rather than a must-do.  It would be more ideal if this could be a call to toObject(), which is more obviously "circomlibjs's way of converting buffers to numbers for public consumption".  However toObject() explicitly doesn't work for packed points because it tries to convert from Montgomery form, and that's already been done during packing.

Reply:
Correct me if I'm wrong but I remember there was no conversion to/from Montgomery in `circomlibjs@0.0.8`.

I think it's still worth using `Scalar.fromRprLE` to really test if the two libraries match their outputs with *their* functions.

Reply:
I actually used: `leInt2Buff`, to match what `circomlibjs` uses here: https://github.com/iden3/circomlibjs/blob/c71c8c1daf54d76db8e0858606faa017783df6a1/src/babyjub.js#L97.

Let me know if it makes sense!

Reply:
Great point.  All my digging into circomlibjs code has been based on the latest version, so needs to be adjusted for the different version you're using.